docs/e2e-test.md

```diff
@@ -106,13 +106,34 @@ in the ephemeral cluster either before pivoting or after re-pivoting.
 
 ### K8s upgrade tests
 
-Kubernetes version upgrade in target nodes. E2E has following k8s version
-upgrades:
+Kubernetes version upgrade in target nodes. We run three latest version
+upgrades in k8s-upgrade tests for main branch. And one kubernetes upgrade
```

Comment:
```suggestion
upgrades in k8s-upgrade tests for main branch and one kubernetes upgrade
```

Reply:
Thanks for the review. Addressed the change request